infra/feast-operator/internal/controller/services/services.go

```diff
@@ -371,6 +422,27 @@ func (feast *FeastServices) setContainer(containers *[]corev1.Container, feastTy
 	*containers = append(*containers, *container)
 }
 
+func (feast *FeastServices) setRoute(route *routev1.Route, feastType FeastServiceType) error {
+	svcName := feast.GetFeastServiceName(feastType)
+	route.Labels = feast.getFeastTypeLabels(feastType)
+
+	tls := feast.getTlsConfigs(feastType)
+	route.Spec = routev1.RouteSpec{
+		To: routev1.RouteTargetReference{
+			Kind: "Service",
+			Name: svcName,
+		},
+		Port: &routev1.RoutePort{
+			TargetPort: intstr.FromInt(int(getTargetPort(feastType, tls))),
+		},
+		TLS: &routev1.TLSConfig{
+			Termination: routev1.TLSTerminationPassthrough,
+		},
```

Comment:
shouldn't this be conditional ? depending on if `services.ui.tls` is being used?

Reply:
also what about http vs https for the route url? are there other route configs to consider?